apps/docs/content/guides/database/connection-management.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: 'connection-management'
+title: 'Connection management'
+description: 'Managing connections'
+subtitle: 'Using your connections resourcefully'
+---
+
+## Connections
+
+Every [Compute Add-On](/docs/guides/platform/compute-add-ons) has a pre-configured direct connection count and Supavisor pool size. This guide discusses ways to observe and manage them resourcefully.
+
+### Configuring Supavisor's pool size
+
+You can change how many database connections Supavisor can manage by altering the pool size in the "Connection pooling configuration" section of the [Database Settings](/dashboard/project/_/settings/database):
+
+![Connection Info and Certificate.](/docs/img/database/pool-size.png)
+
+The general rule is that if you are heavily using the PostgREST database API, you should be conscientious about raising your pool size past 40%. Otherwise, you can commit 80% to the pool. This leaves adequate room for the Authentication server and other utilities.
```

Comment:
I think we're calling it Data API right? Can still reference PostgREST.

```suggestion
The general rule is that if you are heavily using the Data API (PostgREST), you should be conscientious about raising your pool size past 40%. Otherwise, you can commit 80% to the pool. This leaves adequate room for the Authentication server and other utilities.
```

Are we sure we can make this recommendation? I'm not sure if it does leave adequate room and highly depends on the compute instance.

I did an analysis here a while back: https://www.notion.so/supabase/Self-serve-database-connection-management-02248df94cee438aa87ba33e43b4e904?pvs=4#8f2c4df699744348b64ffc91bde3d434

Reply:
The suggestion is based on a [comment](https://supabase.slack.com/archives/C04JR9DBNQL/p1703625889822769?thread_ts=1703625707.918479&cid=C04JR9DBNQL) from @chasers. We can omit the suggestion, but I've found that it leads to indecision by developers. They need a concrete starting place. 

PostgREST uses a dynamic pooler that closes idle connections after 30s and will pend queries for up to 10s. Auth doesn't have a pooler and instead immediately closes connections after the request completes. 

Unless someone has sustained peaks in connection usage, it's unlikely that a collision will occur, even if they do overcommit. 

We can be more conservative with the recommendation and change to 70%